### ISSUE TITLE
不完全适合我

### ISSUE THREAD

**Issue by ksorah1** [x-nas/WinPacketsEdit#17]
早上好，

感谢您的软件，一切都比其他软件好，但是当我尝试使用过滤器或手动发送时，没有任何反应。

**Comment by ksorah1** [comment]
sad

**Comment by x-nas** [comment]
可以截图发一下看看吗，或者看下日志里面是否有报错信息

**Comment by ksorah1** [comment]
hello, I use your software without knowing Chinese, I can do it because I have known wpe pro for 15 years, but I do not know where your logs are. Could you direct me please?
当前内核版本：1.0.8767.26688
开始注入目标进程 =>> software
目标进程是64位程序，已自动调用64位的注入模块!
已成功注入目标进程 =>> software[2156]
注入完成，可关闭当前注入器.

**Comment by ksorah1** [comment]
![2024-01-19 02_20_46](https://github.com/x-nas/WinPacketsEdit/assets/112913359/29e43908-82f1-4583-b825-658bfcb81bcb)
![2024-01-19 02_19_42](https://github.com/x-nas/WinPacketsEdit/assets/112913359/0490d3cc-3c55-4fd1-b93c-d7528e0eab74)
in red : is this to change the socket id?

如果你愿意的话我可以翻译

**Comment by ksorah1** [comment]
![2024-01-19 02_20_46](https://github.com/x-nas/WinPacketsEdit/assets/112913359/67155504-cb7a-478d-8f07-aae7194eec47)

**Comment by ksorah1** [comment]
![Uploading 2024-01-19 02_19_22.jpg…]()

**Comment by ksorah1** [comment]
![2024-01-19 02_19_22](https://github.com/x-nas/WinPacketsEdit/assets/112913359/b6a5c1e5-a612-4dee-b6c3-3a3c11a7d716)

**Comment by ksorah1** [comment]
![2024-01-19 02_19_22](https://github.com/x-nas/WinPacketsEdit/assets/112913359/d2ff5fd2-0b7f-447a-b690-828b5efd67b7)

**Comment by ksorah1** [comment]
![Uploading 2024-01-19 02_19_22.jpg…]()

**Comment by ksorah1** [comment]
![2024-01-19 02_19_22](https://github.com/x-nas/WinPacketsEdit/assets/112913359/326977c6-e2e0-4cf8-a08b-bbd0168dc9d5)

**Comment by ksorah1** [comment]
![2024-01-19 02_19_22](https://github.com/x-nas/WinPacketsEdit/assets/112913359/d8af4db9-0610-4e07-aa47-a416660cb504)

**Comment by ksorah1** [comment]
![2024-01-19 02_19_22](https://github.com/x-nas/WinPacketsEdit/assets/112913359/f88385a5-d442-4e38-949f-2a314d4e9f92)

**Comment by x-nas** [comment]
> ![2024-01-19 02_19_42](https://private-user-images.githubusercontent.com/112913359/297924175-0490d3cc-3c55-4fd1-b93c-d7528e0eab74.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU4ODc5NDAsIm5iZiI6MTcwNTg4NzY0MCwicGF0aCI6Ii8xMTI5MTMzNTkvMjk3OTI0MTc1LTA0OTBkM2NjLTNjNTUtNGZkMS1iOTNjLWQ3NTI4ZTBlYWI3NC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyMlQwMTQwNDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYzdjNGFiMTg5Nzc5ZGY2NzMyNWYyODA3NjZiNTBlM2YwNGM0ODljMDY3YmI2MTlkZjgyOWQwZGNhMjFlZjFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Pg1izOlankCCnxfb80eB9GaovitryDkJ8RYlKI1SWvA) in red : is this to change the socket id?
> 
> 如果你愿意的话我可以翻译


Yes, that means using this specific socket ID to send all packets

**Comment by x-nas** [comment]
![111](https://github.com/x-nas/WinPacketsEdit/assets/67667226/a8953dbe-61f2-4f5c-b7bf-2173b1a27c0e)

Hope this can help you

**Comment by ksorah1** [comment]
Thanks for your help ! Indeed the "send"
is an error, I will test a little more. But strangely the filter doesn't work either, here's how I use it (in censored white, these are the same offsets):
![2024-01-23 15_47_12](https://github.com/x-nas/WinPacketsEdit/assets/112913359/b6da2812-c7b7-420b-9334-9ced8553f755)
![2024-01-23 15_46_33](https://github.com/x-nas/WinPacketsEdit/assets/112913359/d3f6fb8d-60e6-45d8-8d7f-d41e8fffe60e)

**Comment by ksorah1** [comment]
Thanks for your help ! Indeed the "send"
is an error, I will test a little more. But strangely the filter doesn't work either, here's how I use it (in censored white, these are the same offsets):
![2024-01-23 15_47_12](https://github.com/x-nas/WinPacketsEdit/assets/112913359/f1b4a4fe-d07f-4d39-b2f7-de1fbcbfb42f)
![2024-01-23 15_46_33](https://github.com/x-nas/WinPacketsEdit/assets/112913359/15bfae74-9907-4a6c-b181-3a51174bcf81)
